### ISSUE TITLE
Check only a single nav exists OR multiple navs have aria labels

### ISSUE THREAD

**Issue by danspratling** [jackdomleo7/Checka11y.css#1]
Only a single navigation element is permitted. This is because `<nav>` dictates the *primary* navigation for the page (often the page header).

Where multiple navigations exist on a page (for example, the footer), any which aren't considered *primary* should not be wrapped in a `<nav>` tag.

Evidence: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav
(See usage notes)

**Comment by danspratling** [comment]
After discussion it had been noted that multiple navs may exist at once, but if they do should be labelled correctly.

Todo:
- [] adjust the test to ensure many navs may be present but if they are then "aria-label" or "aria-labelledby" must be defined and not empty on both elements.
- [] update error description to clearly explain this
